### ISSUE TITLE
Missing content value 'LockedToProjectWithoutNested' for content_permissions when creating a project

### ISSUE THREAD

**Issue by codenametableau** [tableau/server-client-python#840]
**Describe the bug**
Missing content value 'LockedToProjectWithoutNested' for content_permissions when creating a project. This value is available via rest api but not tsc.

**Versions**
Details of your environment, including:
 - Tableau Server version: 2020.3.5
 - Python version: 3.5

**To Reproduce**
```
def create_project(server, project_item):
    try:
        project_item = server.projects.create(project_item)
        print('Created a new project called: %s' % project_item.name)
        return project_item
    except TSC.ServerResponseError:
        print('We have already created this project: %s' % project_item.name)
        sys.exit(1)

top_level_project = TSC.ProjectItem(name='Name', content_permissions='LockedToProjectWithoutNested')
top_level_project = create_project(server, top_level_project)

```

**Results**
```
ValueError: Invalid value: LockedToProjectWithoutNested. content_permissions must be of type ContentPermissions.
```

**Comment by elsherif** [comment]
Hello,

I'm facing same problem, any idea when that would be fixed?

Thanks a lot
Saad

**Comment by ovinis** [comment]
This issue has been fixed in TSC version 0.16.0. Please upgrade to the new version - [New version link](https://pypi.org/project/tableauserverclient/0.16.0/)
